### PR TITLE
Add secondary sidebar section and image

### PR DESCRIPTION
**Pull request by cadamini**
Fixes https://github.com/microsoft/vscode-docs/issues/6326

Today I learned about the Secondary Side Bar - thanks! Here's some documentation. I hope you like it.

### CODE REVIEW

File: docs/getstarted/userinterface.md [hunk 1]
```diff
@@ -53,6 +53,17 @@ When you have more than one editor open you can switch between them quickly by h
 
 >**Tip:** You can resize editors and reorder them. Drag and drop the editor title area to reposition or resize the editor.
 
+## Secondary Side Bar
+
+A Secondary Side Bar is also available to display views opposite the Primary Side Bar. You can open it by using one of the following ways:
```
Comment (cadamini) [inline docs/getstarted/userinterface.md:58]:
```suggestion
The Secondary Side Bar can display views opposite the Primary Side Bar. You can open it as follows:
```

File: docs/getstarted/userinterface.md [hunk 2]
```diff
@@ -53,6 +53,17 @@ When you have more than one editor open you can switch between them quickly by h
...
+
+- Use the keyboard shortcut `kb(workbench.action.toggleAuxiliaryBar)`.
+- Open the [Command Palette](/docs/getstarted/userinterface.md#command-palette) with the keyboard shortcut `kb(workbench.action.showCommands)` and run the `View: Toggle Secondary Side Bar Visibility` command.
+
+You can drag Activity Bar icons onto the new Secondary Side Bar that has appeared opposite your Primary Side Bar. Create more tabs by dragging more Activity Bar icons into the top tab area of the Secondary Side Bar.
```
Comment (cadamini) [inline docs/getstarted/userinterface.md:63]:
```suggestion
You can drag Activity Bar icons onto the new Secondary Side Bar that has appeared opposite your Primary Side Bar. Create more tabs by dragging more icons or views into the top tab area of the Secondary Side Bar. 

> **Tip:** Tip You can open all available views using the `View: Open View` command or the **View** > **Open View** menu command.
```

File: docs/getstarted/userinterface.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ VS Code comes with a simple and intuitive layout that maximizes the space provid
 * **Activity Bar** -  Located on the far left-hand side, this lets you switch between views and gives you additional context-specific indicators, like the number of outgoing changes when Git is enabled.
 * **Panel** - An additional space for views below the editor region. By default, it houses output, debug information, errors and warnings, and an integrated terminal. Panel can also be moved to the left or right for more vertical space.
 
->**Tip:** A Secondary Side Bar is also available to display views opposite the Primary Side Bar. You can show it with `kb(workbench.action.toggleAuxiliaryBar)`.
+>**Tip:** A [Secondary Side Bar](/docs/getstarted/userinterface.md#secondary-side-bar) is also available to display views opposite the Primary Side Bar.
```
Comment (cadamini) [inline docs/getstarted/userinterface.md:26]:
```suggestion
>**Tip:** A Secondary Side Bar is also available to display views opposite the Primary Side Bar. Learn more on how to open it and drag elements into it in the [Secondary Side Bar](/docs/getstarted/userinterface.md#secondary-side-bar) section.
```

File: docs/getstarted/userinterface.md [hunk 4]
```diff
@@ -53,6 +53,19 @@ When you have more than one editor open you can switch between them quickly by h
 
 >**Tip:** You can resize editors and reorder them. Drag and drop the editor title area to reposition or resize the editor.
 
+## Secondary Side Bar
+
+The Secondary Side Bar can display views opposite the Primary Side Bar. You can open it as follows:
+
+- Use the keyboard shortcut `kb(workbench.action.toggleAuxiliaryBar)`.
+- Open the [Command Palette](/docs/getstarted/userinterface.md#command-palette) with the keyboard shortcut `kb(workbench.action.showCommands)` and run the `View: Toggle Secondary Side Bar Visibility` command.
```
Comment (gregvanl) [inline docs/getstarted/userinterface.md:61]: The convention is to use bold for commands so **View: Toggle Secondary Side Bar Visibility**

File: docs/getstarted/userinterface.md [hunk 5]
```diff
@@ -53,6 +53,19 @@ When you have more than one editor open you can switch between them quickly by h
...
+
+You can drag Activity Bar icons onto the new Secondary Side Bar that has appeared opposite your Primary Side Bar. Create more tabs by dragging more icons or views into the top tab area of the Secondary Side Bar. 
+
+> **Tip:** Tip You can open all available views using the `View: Open View` command or the **View** > **Open View** menu command.
```
Comment (gregvanl) [inline docs/getstarted/userinterface.md:65]: bold commands so **View: Open View**

File: docs/getstarted/userinterface.md (hunk 4 above)
Comment (cadamini) [reply docs/getstarted/userinterface.md:61]:
```suggestion
- Open the [Command Palette](/docs/getstarted/userinterface.md#command-palette) with the keyboard shortcut `kb(workbench.action.showCommands)` and run the **View: Toggle Secondary Side Bar Visibility** command.
```

Thanks for the hint!

File: docs/getstarted/userinterface.md (hunk 5 above)
Comment (cadamini) [reply docs/getstarted/userinterface.md:65]:
```suggestion
> **Tip:** Tip You can open all available views using the `View: Open View` command or the **View** > Open View** menu command.
```

Oups, formatting mistake. Thanks. ✨

**Comment by cadamini** [comment]
FYI - Not sure how to get this merged. I will close my PR in a week.

**Comment by gregvanl** [comment]
Hi @cadamini Thanks for putting together this PR.

Per comment in https://github.com/microsoft/vscode-docs/issues/6326, not sure we need more documentation in the Get Started > User Interface topic.

Might be a good addition to an "Advanced Editing" topic mentioned in https://github.com/microsoft/vscode-docs/pull/6337#issuecomment-1585039588

Closing for now (PR is also missing an image).

**Comment by cadamini** [comment]
Any plans to start the Advanced Editing topic?

**Pull request closed without merging**